### PR TITLE
Style changes for metrics docs

### PR DESCRIPTION
**Pull request by sivasamyk**
Few of the metrics tables are reordered based on the definitions in YAML and alphabetical sorting of YAML files. For metrics without descriptions (Hadoop & Spark) find the screenshot attached for display.
screen shot 2018-05-28 at 7 06 32 pm

### CODE REVIEW

File: docs/integration/cassandra.md [hunk 1]
```diff
@@ -4,48 +4,48 @@
...
-avg local read latency      |  cassandra.local.read.latency.avg                   |  gauge        |  double        |  ms                                 |
+Metric Name<br> Key *(Type)* *(Unit)*                                                                     |  Description
+----------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------
+leaving nodes<br>**cassandra.cluster.nodes.leaving** <br>*(long gauge)*                                   |
+moving nodes<br>**cassandra.cluster.nodes.moving** <br>*(long gauge)*                                     |
+joining nodes<br>**cassandra.cluster.nodes.joining** <br>*(long gauge)*                                   |
+live nodes<br>**cassandra.cluster.nodes.live** <br>*(long gauge)* *(from the perspective of this node)*   |  Count of nodes which are visible and live
+null<br>**cassandra.cluster.nodes.unreachable** <br>*(long gauge)*                                        |
```
Comment (bsmid) [inline docs/integration/cassandra.md]: "null"? I see there are few more "null"s in other places.

File: docs/integration/cassandra.md [hunk 2]
```diff
@@ -4,48 +4,48 @@
...
-avg local write latency     |  cassandra.local.write.latency.avg                  |  gauge        |  double        |  ms                                 |
-local read request          |  cassandra.local.read.requests                      |  counter      |  long          |                                     |  Local reads retrieve data from a table's memtable and any necessary SSTables on disk
-local read latency total    |  cassandra.local.read.requests.latency.time         |  counter      |  long          |  ms                                 |
-avg local read latency      |  cassandra.local.read.latency.avg                   |  gauge        |  double        |  ms                                 |
+Metric Name<br> Key *(Type)* *(Unit)*                                                                     |  Description
+----------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------
+leaving nodes<br>**cassandra.cluster.nodes.leaving** <br>*(long gauge)*                                   |
```
Comment (bsmid) [inline docs/integration/cassandra.md]: Should "Long" be lowercased?

File: docs/integration/cassandra.md (hunk 1 above)
Comment (sivasamyk) [reply docs/integration/cassandra.md]: fixed in GenerateDocs code. looks like missed replacing updated metrics. Will regenerate docs

File: docs/integration/cassandra.md (hunk 2 above)
Comment (sivasamyk) [reply docs/integration/cassandra.md]: it was lower case when displayed numeric type in a separate column. now we have combined numeric type and metric type. e.g `Long gauge` `Double counter` is fine?

Comment (bsmid) [reply docs/integration/cassandra.md]: For some reason screenshot is showing lowercase, but source has uppercase? I prefer lowercased type, but it is ok.

**Pull request merged**